packages/ucp-enforcement/src/Explanation.ts

```diff
@@ -0,0 +1,148 @@
+import { Store, DataFactory } from "n3";
```

Comment:
Haven't looked into depth here, but not sure how to interpret the output. I ran the example script with `calculateAndExplainAccessModes`, but the only information I get (apart from the request and the access modes) is `algorithm: 'Union'`.